app/helpers/helpers/helpers_file.go

```diff
@@ -8,6 +8,39 @@ import (
 	"strings"
 )
 
+func GetCommand(line string) string {
```

Comment:
please add a comment with all the examples it covers
it is not clear what this function does

Reply:
addressed in d9c081de52e70f86c4ee0af1d15f3af69343fabf

Reply:
let us review together the comment....